src/compliment/utils.clj

```diff
@@ -241,3 +243,32 @@ Note that should always have the same value, regardless of OS."
         ;; resource pathes always use "/" regardless of platform
         (.. (ensure-no-leading-slash file)
             (replace File/separator resource-separator))))))
+
+(defn var->class
+  "Given a form that may be a var, returns the class that is associated to its :tag or its value (in that precedence order)."
+  [ns form]
+  (when-let [var-ref (and (symbol? form)
+                          (when-let [found (some->> form (ns-resolve ns))]
```

Comment:
Not sure why you need `some->>` here.
Same for `when-let`, you check for `var?` later, it handles `nil` just fine.